docsk8s/troubleshooting/ls-k8s-troubleshooting-methods.asciidoc

```diff
@@ -7,6 +7,8 @@ Here are some approaches that you can use to diagnose the state of your {ls} and
 * <<ls-k8s-viewing-logs>>
 * <<ls-k8s-connecting-to-a-container>>
 * <<ls-k8s-diagnostics>>
+* <<ls-k8s-pq-util>>
+* <<ls-k8s-pq-drain>>
```

Comment:
In a future revision, let's look at grouping the PQ-related items into one category.